addons/website_forum/models/forum.py

```diff
@@ -860,46 +864,66 @@ def _get_karma_value(self, old_vote, new_vote, up_karma, down_karma):
 
     @api.model
     def create(self, vals):
+        # can't modify owner of a vote
+        if self._uid != SUPERUSER_ID:
```

Comment:
are you sure, since we add support of superuser and admin ?

Reply:
Indeed, good catch, I simply rebased on 12.1 but this had to be replaced by `_is_admin()`.
Note that other occurence of SUPERUSER_ID wrongly appear in website_forum (I added the fix in this PR and I will backport it + fix others (website_slides) at some point).